test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/ObjectExplorerServiceTests.cs

```diff
@@ -243,6 +243,26 @@ public ObjectExplorerServiceTests()
             connectionServiceMock.Verify(c => c.Disconnect(It.IsAny<DisconnectParams>()));
         }
 
+        [Fact]

```

Comment:
please add test for finding column. basically verifying all different options in FindNoresRequest